app/scripts/modules/netflix/availability/availability.read.service.ts

```diff
@@ -0,0 +1,57 @@
+import { module } from 'angular';
+
+import {API_SERVICE, Api} from 'core/api/api.service';
+
+export interface IAvailabilityDataTimeSeries {
+  is_outage: number[];
+  availability: number[];
+  datetime: string[];
+}
+
+export interface IAvailabilityWindow {
+  availability: number;
+  nines: number;
+  target_nines: number;
+  date_range: Date[];
+  ts: IAvailabilityDataTimeSeries;
+  score?: number;
+  [key: string]: any;
+}
+
+export interface IAvailabilityData {
+  last_updated: string;
+  yesterday: IAvailabilityWindow;
+  '28days': IAvailabilityWindow;
```

Comment:
i know why it was done, but i'm not crazy about having fields start with numbers.  especially since the corresponding fields on `$scope` are spelled out.